Added debug output for non-silent mode

tjoskar opened the pull request:
Hi,

I think it would be nice to print out some debuging info like requested `url`, `verb`, `headers` and `body`.
Since it doesn't seems you are using the silent option I used that option for the debug print.

Commit:
Added debug output for non-silent mode

If the silent-flag is set to false the server will print some debug info like
request-method, -headers, -body and the requested url

jhudson8 commented:
I'm fine with adding a flag like this but I don't want it to be the default.  How about you change it to an opt-in `verbose` flag.

Commit:
Added verbose option for printing request data

Added verbose option and changed the option for printing incoming
request data from silent to verbose.

Commit:
Removed tern datafile

I accidentally added a tern-file in the last commit, I'am sorry for that.

tjoskar commented:
Sure, I added an opt-in `verbose` option.
